torch/_functorch/aot_autograd.py

```diff
@@ -30,6 +30,8 @@
 from .partitioners import default_partition
 from torch._guards import TracingContext, DuplicateInputs
 
+guard__ = torch.autograd._force_original_view_tracking(True)
```

Comment:
I guess this is a debug statement? This will enable original view tracking as soon as this file is imported.

Reply:
ha yep, this was me debugging. Thanks!